plugins/destination/elasticsearch/client/spec.go

```diff
@@ -1,31 +1,54 @@
 package client
 
-import "runtime"
+import (
+	_ "embed"
+	"runtime"
+)
 
 const (
 	defaultBatchSize      = 1000
 	defaultBatchSizeBytes = 5 * 1024 * 1024
 )
 
 type Spec struct {
-	Addresses []string `json:"addresses"` // A list of Elasticsearch nodes to use.
-	Username  string   `json:"username"`  // Username for HTTP Basic Authentication.
-	Password  string   `json:"password"`  // Password for HTTP Basic Authentication.
+	// A list of Elasticsearch nodes to use. (Default: ["http://localhost:9200"])
```

Comment:
No need for default repetition here.
```suggestion
	// A list of Elasticsearch nodes to use.
```